app/settings.py

```diff
@@ -139,7 +140,7 @@ def utcoffset_or_fail(date_value, key):
 )
 
 VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS = int(
-    os.getenv("VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS", "2700")
+    os.getenv("VIEW_SUBMITTED_RESPONSE_EXPIRATION_IN_SECONDS", "10")
```

Comment:
This needs reverting.

Reply:
Accidentally changed, was testing something else